Sources/BraintreeDropIn/BTPaymentSelectionViewController.h

```diff
@@ -38,6 +38,8 @@
 /// @param error The error that occured during tokenization of a new payment method.
 - (void) selectionCompletedWithPaymentMethodType:(BTUIKPaymentOptionType) type nonce:(BTPaymentMethodNonce *)nonce error:(NSError *)error;
 
+- (void)cancelButtonPressedOnPaymentSelectionViewController:(BTPaymentSelectionViewController *)viewController;
```

Comment:
Tell `BTDropInController` that the cancel button was tapped on the payment selection screen.

Reply:
Docstring could be helpful here, for internal reference!

Reply:
✔️